Update pipes-and-filters-content.md (#4475)
MicrosoftDocs/architecture-center: docs/patterns/pipes-and-filters-content.md

Consider the following points when you decide how to implement this pattern:

> - **Monolithic nature**. This pattern is usually implemented as a monolithic deployment, so for any change, the entire filter chain has to be tested and deployed. Also, fault-tolerance for the whole process isn't well supported; if a filter or pipe fails, the whole pipeline is likely to fail.

Reviewer: ```suggestion
- **Monolithic nature**. This pattern is usually implemented as a monolithic pipeline, so for any change, the entire filter chain should be tested end to end. Also, fault-tolerance for the whole process needs to be considered; if a filter or pipe fails, the whole pipeline is likely to fail.
```
